src/Pages/Reconstruction.jsx

```diff
@@ -88,17 +134,20 @@ export class Reconstruction extends Component {
         {/* left-section */}
         <div className='col-6'>
           {/* form  */}
-          <form id='form' onSubmit={this.submitHandler} className='form-group'>
+          <form
+            ref={this.form}
+            onSubmit={this.submitHandler}
+            className={`form-group ${
+              (this.state.uploading || this.state.uploadSuccess) && 'd-none'
```

Comment:
Else if the condition evaluates to false, we will have `false` as class in the form element.
```suggestion
              (this.state.uploading || this.state.uploadSuccess) ? 'd-none' : ''
```